### DISCUSSION TITLE
TyC Sports

### DISCUSSION THREAD

**Discussion by arkina1234-undenified** [iptv-org/community#364] [Show and tell]
### Channel ID

TyCSports.ar

### Stream URL

https://d3awnlgqz0szay.cloudfront.net:443/out/v1/b841c366cbe540e6a106c3ba83e5c8d6/index.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Thank you for the stream; it will be included in the next ar.m3u update.

**Comment by PepeParada** [comment]
gracias. funciona!

**Comment by santi0958** [comment]
Alguno sabe como verlo en la computadora?
-----------
Guys, does anyone know if this link works in the computer or only cellphone?

**Comment by AntiPontifex** [comment]
afaik, it should work it both

**Comment by LucasRao13** [comment]
the link doesn't work... help

**Comment by sdefilippe** [comment]
Link doesn’t work!!

**Comment by LucasRao13** [comment]
Help pls bro

**Comment by gabrielcarp9** [comment]
> Help pls bro

I have a link but is dash (.mpd). can you reproduce it?

**Comment by LucasRao13** [comment]
Send me pls @gabrielcarp9

**Comment by gabrielcarp9** [comment]
> Send me pls @gabrielcarp9
The link is: https://is-tucuman.cdn.rcs.net.ar/mnp/tyc/output.mpd
Reproduce with Slix in phone or VLC in PC

**Comment by AntiPontifex** [comment]
Where I am (Canada); the feed only lasts for a few seconds, then stops. Thanks for posting, however.

**Comment by gabrielcarp9** [comment]
> Where I am (Canada); the feed only lasts for a few seconds, then stops. Thanks for posting, however.

Do you have discord?

**Comment by ovagrim** [comment]
Ponerlo otra vez please.

**Comment by LucasRao13** [comment]
me too, if you can solve this problem, can you help me by commenting how you did it?

**Comment by Nicolas1307** [comment]
No funciona ponerlo devuelta

**Comment by AntiPontifex** [comment]
@gabrielcarp9 - Listed in the Discussion section in this github, is a topic about Discord. In it, is a link to a channel for this group: hope to talk to you there.

**Comment by thetinean** [comment]
> @gabrielcarp9 - Listed in the Discussion section in this github, is a topic about Discord. In it, is a link to a channel for this group: hope to talk to you there.

Unfortunately the Discord link on that post is no longer valid. I am also looking for some info on this channel, do you have a new link for this server? Thank you.

**Comment by AntiPontifex** [comment]
@gabrielcarp9 - Try this: https://discord.gg/QrDAj4WK

**Comment by PepeParada** [comment]
**Otra vez por favor!! hoy juega Argentina!!**

Once again please!! Argentina is playing today!!

**Comment by freearhey** [comment]
https://d3awnlgqz0szay.cloudfront.net/out/v1/b841c366cbe540e6a106c3ba83e5c8d6/index.m3u8 (offline)
<img width="716" alt="image" src="https://user-images.githubusercontent.com/7253922/232244941-9b5d5cd4-5acf-4df7-9b68-ddd80b4d0b06.png">
